boards/shields/reel_board_canbus/reel_board_canbus.conf

```diff
@@ -0,0 +1,3 @@
+CONFIG_SPI=y
```

Comment:
Shield designation is "link board can", could you rename everything to `link_board_can`*, please?

Reply:
Sure